packages/react-core/src/next/components/Wizard/WizardContext.tsx

```diff
@@ -68,35 +62,52 @@ export const WizardContextProvider: React.FunctionComponent<WizardContextProvide
   const [currentFooter, setCurrentFooter] = React.useState(
     typeof initialFooter !== 'function' ? initialFooter : undefined
   );
-  const currentStep = getCurrentStep(steps, currentStepIndex);
 
-  const goToNextStep = React.useCallback(() => onNext(steps), [onNext, steps]);
-  const goToPrevStep = React.useCallback(() => onBack(steps), [onBack, steps]);
+  // Combined initial and current state steps
```

Comment:
Cool application of `useMemo` here! 

If this is just an array of all the steps, I'm wondering if `allSteps` or just `steps` might be a better name - it looks like this was just called `steps` initially? I wouldn't block over this, but it might help to have some more context here!

Reply:
That's a good suggestion. I agree, 'merged' doesn't tell much of a story. I'll update it to just be `steps`.